core/src/main/java/tc/oc/pgm/observers/tools/NightVisionTool.java

```diff
@@ -17,6 +17,12 @@
 
 public class NightVisionTool implements InventoryMenuItem {
 
+  final InventoryMenu in;
+
+  public NightVisionTool(InventoryMenu in) {
+    this.in = in;
```

Comment:
```suggestion
  final InventoryMenu menu;

  public NightVisionTool(InventoryMenu menu) {
    this.menu = menu;
```

Reply:
Wops this is just code I forgot to remove